builder/profiles/fastboot.cmake

```diff
@@ -34,3 +34,5 @@ set( MFX_ENABLE_H264_VIDEO_ENCODE OFF )
 set( MFX_ENABLE_H264_VIDEO_FEI_ENCODE OFF )
 set( MFX_ENABLE_H265_VIDEO_ENCODE OFF )
 set( MFX_ENABLE_HEVC_VIDEO_FEI_ENCODE OFF )
+set ( MFX_ENABLE_MPEG2_VIDEO_DECODE ON)
```

Comment:
By the way, it makes sense to list those components which you enable _together_. For example in the very end of the file. But, please, don't change anything for the BUILD_* settings.